<div class="alert alert-block alert-danger">

# HT for One Proportion (COMPLETE)
    
</div>

Made in collaboration with [Skew the Script](https://skewthescript.org/) and [CourseKata](https://coursekata.org/).

<img src="https://i.postimg.cc/ty1GkxB8/Skew-the-Script-Logo.png" title="Skew the script logo" width=200 align = left>

<img src="https://i.postimg.cc/tXcF0nzD/Course-Kata-logo.png" title="CourseKata logo" width=200 align = right>

In [ ]:
# Load the CourseKata library
suppressPackageStartupMessages({
    library(coursekata)
})

### 1.0 - The Flint Water Crisis

In 2014, city officials in Flint, Michigan, celebrated the cost-cutting measure of switching the city's water source from Detroit to the local Flint River. By 2015, residents of Flint began to report problems with the water (e.g., discolored water, foul smelling water, health problems) and complained that the water was unsafe. Researchers claimed that the new water from the Flint River corrodes pipes, so lead gets into the water. Among many other side effects, ingesting lead can hurt children’s brain development. As a rebuttal, the mayor of Flint drank a cup of tap water on television as a way to demonstrate that the water was safe. However, residents continued to insist that something was wrong, and outside investigators began testing the water.

<img src="https://i.postimg.cc/D2Y0F3gz/HT-1-Prop-Flint-Water-Crisis-Timeline.png" title="Flint Water Crisis Timeline" width = 800/>

**Today’s Key Analysis**

Was there convincing evidence that the water was unsafe?

### 2.0 - The Data

According to EPA* regulations, if more than 10% of homes in a city have high lead content in their water
(>15 parts per billion), the city’s water supply is unsafe.

To help us determine whether there is convincing statistical evidence that the Flint water system is unsafe, we will refer to data from a Virginia Tech study**. They randomly sampled water from 252 homes in Flint. Of those, 42 had high lead content.

*\*Info from [Environmental Protection Agency](https://www.epa.gov/ground-water-and-drinking-water/basic-information-about-lead-drinking-water) (EPA) website article, “Basic Information about Lead in Drinking Water.” (accessed 7/22/2020)*

*\*\*[Edwards, M., et al.](http://flintwaterstudy.org/information-for-flint-residents/results-for-citizen-testing-for-lead-300-kits/) Virginia Tech’s Flint Water Study (published Sept. 2015, accessed July 2020)*

**2.1:** Using this data, find $\hat p$.

In [ ]:
# Find the sample proportion of homes with high lead content

# COMPLETE VERSION
42 / 252

[1] 0.1666667

### 3.0 - Thinking about the DGP

Of course, we are ultimately interested in the DGP. What does our empty, or null, model assume? How about the alternative model?

**3.1:** Which statements below map onto the hypothesis of the empty model? Which statements map onto the alternative model?

> - Flint's water is safe to drink

> - $p <= .10$

> - Flint's water is not safe to drink

> - $p > .10$

<div class="alert alert-block alert-warning">

**Sample Response**

#### Empty Model

- Flint's water is safe to drink

- $p <= .10$

#### Alternative Model

- Flint's water is not safe to drink

- $p > .10$

</div>

#### A DGP where $p = .10$

We can start by assuming that the empty model is true, and that $p$, the true proportion of all homes in Flint with high lead content in their water, is 10%:

<img src="https://i.postimg.cc/nn3D87N4/HT-1-Prop-P-of-10-percent.png" title="P of Ten Percent" width = 600/>

Next, we ask: If the empty model is in fact true, how likely is the data we've gathered? To help us, we will need a sampling distribution.

The Flint study collected water from 252 homes (n = 252). At this sample size, what estimates for $p$ are we likely to get? Let’s find the sampling distribution of $\hat p$ from all possible random samples.

##### Let's plug in what we know ($n$, $p_0$) and do our calculations:

<img src="https://i.postimg.cc/8GnR1jPT/HT-1-Prop-Calc-SDo-Prop.png" title="Calculate SDoP" width = 800/>

##### Compare the sample proportion, $\hat p = 16.7\%$, to the resulting sampling distribution:

<img src="https://i.postimg.cc/ZbBt1Cjw/HT-1-Prop-SDo-P-and-the-Data.png" title="SDoP and the Data" width = 800/>

### 4.0 - How unlikely is this data?

We can see that our sample proportion falls in the upper tail of the distribution. If we use the Empirical Rule (the 68 – 95 – 99.7 Rule), it suggests that it is pretty unlikely (less than 5% chance) to get a sample proportion more than two standard deviations out from the center.

How do we figure out how many standard deviations away from the center our sample is?

#### From Z-Scores to Z-Test Statistics

A z-score tells us how many standard deviations a value is above or below the mean (or the empty model).

$z = \frac {data value - mean}{standard deviation}$

We can extend this idea to our sample statistics andjust take the distance between the sample proprtion and the empty model and divide it by the standard deviation (or standard error) to see how many standard deviations fit in that distance.

$z = \frac {\hat p - \mu_{\hat p}}{\sigma_{\hat p}}$

##### How many standard deviations away from $p_0$ is $\hat p$?

<img src="https://i.postimg.cc/hDn0TY6c/Ht-1-Prop-z-score.png" title="Z Score" width = 800/>

If 2 standard deviations away from $p_0$ would be unlikely, then 3.5 standard deviations away is *very* unlikely.

The larger the test statistic (in magnitude), the more unusual your measurements are under the null
assumption. We should probably start to doubt the $p_0$ assumption.

But exactly how unlikely was the proportion we sampled?

For this, we need p-value!

<img src="https://i.postimg.cc/CSshCr1J/HT-1-Prop-Find-this-Probability.png" title="Find the Probability" width = 800/>

In [ ]:
#find p-value
pnorm(q=3.53, lower.tail=FALSE)

[1] 0.0002077798

### 5.0 - Conclusions

**5.1:** What do you conclude regarding the water safety of Flint?

<div class="alert alert-block alert-warning">

**Sample Response**

Under the assumption that only 10% of homes have high lead content, the actually observed data (16.7% of 252 homes had high lead content) is highly unlikely (p-value = 0.0002). There’s convincing evidence that more than 10% of homes have high lead concentration in their water supply.

</div>

### 6.0 - Discussion

**6.1:** As usual, there is a much quicker way to get our estimates and p-value using R! Run the function `prop.test()` below to calculate the 95% confidence interval for our estimate. Also what other information do you notice that this function calculates for us?

In [ ]:
# Use prop.test() to get the CI
prop.test(x = 42, n = 252, p = 0.1)


	1-sample proportions test with continuity correction

data:  42 out of 252
X-squared = 11.715, df = 1, p-value = 0.0006201
alternative hypothesis: true p is not equal to 0.1
95 percent confidence interval:
 0.1239916 0.2197776
sample estimates:
        p 
0.1666667 


<div class="alert alert-block alert-warning">

**Sample Response**

The confidence interval for the proportion of Flint homes with a high lead level in their water is [0.121, 0.213].

The function also gives info such as the chi-squared value, df, p-value, the 95% CI, and our sample estimate.

</div>


**6.1: Discussion Question:** 

Is this confidence interval consistent with the results of our hypothesis test? Explain.

<img src="https://i.postimg.cc/pVXWBZMh/HT-1-Prop-CI-for-the-sample.png" title="CI for the Sample" width = 600/>


<div class="alert alert-block alert-warning">

**Sample Response**

The interval is consistent with our hypothesis test. The interval of plausible values for the proportion of homes with high lead content (12.1% - 21.3%) does not include our null hypothesis value (10%).

</div>